sdk/search/azure-search/README.md

```diff
@@ -37,28 +37,30 @@ pip install azure-search --pre
 
 ### Create an Azure Cognitive Search service
 
-### Using an Admin Key
+### Using an API Key
 
-Use the [Azure CLI][azure_cli] snippet below to get the Admin Key from the Cognitive Search resource.
+You can get the Query Keys or Admin Key from the resource information in the
+[Azure Portal][azure_portal].
+
+Alternatively, youcan se the [Azure CLI][azure_cli] snippet below to get the
+Admin Key from the Cognitive Search resource.
 
 ```PowerShell
 az search admin-key show --resource-group <your-resource-group-name> --service-name <your-resource-name>
 ```
 
-Alternatively, you can get the endpoint and Admin Key from the resource information in the [Azure Portal][azure_portal].
-
 ### Authenticate the client
 
 Interaction with this service begins with an instance of a [client](#client "search-client").
-To create a client object, you will need the cognitive services or text analytics `endpoint` to
-your resource and a `credential` that allows you access:
+To create a client object, you will need the `search_endpoint` for your resource
```

Comment:
Should be just `endpoint` and not `search_endpoint`, right?

Reply:
fixed